presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -319,6 +319,11 @@ public void testRowSubscript()
 
         // Row subscript in join condition
         assertQuery("SELECT n.name, r.name FROM nation n JOIN region r ON ROW (n.name, n.regionkey)[2] = ROW (r.name, r.regionkey)[2] ORDER BY n.name LIMIT 1", "VALUES ('ALGERIA', 'AFRICA')");
+
+        // Subscript over field named after reserved keyword
+        assertQuery(
```

Comment:
I think we should have a dedicated test for planner (`TestPlanner`?). There is no point to run this test for all the connectors.

Reply:
IIRC the issue occurs when distributing plan fragments to workers, so this must run in distributed mode.